src/pages/docs/deprecations/index.md

```diff
@@ -18,88 +18,92 @@ Deprecations have the following lifecycle:
 
 Further details on the background for this update are available on the [Dropping support for Windows Server 2003 machines](https://octopus.com/blog/deprecating-win2003) blog post.
 
 ### F# Script Steps
 
-Due to the low uptake of F# script steps and the work required to upgrade them for continued use in our modern codebase, we will no longer be supporting F# script steps from `2024.1`. Customers who continue to need F# scripts in later Octopus versions should use standard shell scripting (powershell or bash) and invoke their scripts via their own F# tools included in additional [referenced packages](https://octopus.com/docs/deployments/custom-scripts/run-a-script-step#referencing-packages).
+Due to the low uptake of F# script steps and the work required to upgrade them for continued use in our modern codebase, we will no longer support F# script steps from `2024.1`. Customers who continue to need F# scripts in later Octopus versions should use standard shell scripting (powershell or bash) and invoke their scripts via their own F# tools included in additional [referenced packages](https://octopus.com/docs/deployments/custom-scripts/run-a-script-step#referencing-packages).
 
 ## Deprecations for 2023.3
 
 ### Project level `/runbooks/all` API endpoint {#project-level-runbooks-all-api-endpoint}
 
-The `GET /projects/{projectId}/runbooks/all` API endpoint is being replaced by a new version that omits the ProjectIds query string parameter in future versions of Octopus. It was adopted from an earlier version of the product and is now redundant and potentially confusing.
-The same functionality is available via the `GET /runbooks/all` API endpoint, passing relevant Project IDs via the `ProjectIds` query parameter. If the `ProjectIds` parameter is not required, the `GET /projects/{projectId}/runbooks/all/v2` endpoint should be used instead.
+The `GET /projects/{projectId}/runbooks/all` API endpoint is being replaced by a new version that omits the ProjectIds query string parameter in future versions of Octopus. It was adopted from an earlier product version and is now redundant and potentially confusing.
+The same functionality is available via the `GET /runbooks/all` API endpoint, passing relevant Project IDs via the `ProjectIds` query parameter. If the `ProjectIds` parameter is not required, you can use the `GET /projects/{projectId}/runbooks/all/v2` endpoint.
 
 ### Reporting `/reporting/deployments-counted-by-week` API endpoint {#reporting-deployments-by-week}
 
-The `GET /reporting/deployments-counted-by-week` API endpoint is being removed in future versions of Octopus.
-It is an old endpoint that is no longer used by any of our supported clients.
+The `GET /reporting/deployments-counted-by-week` API endpoint is being removed in future versions of Octopus. None of our supported customers currently use this endpoint.
```

Comment:
the word "clients" in the original implies the software which _calls_ into Octopus - rather than literal customers.